### PR TITLE
fix resource leak on connection refused

### PR DESCRIPTION
**Pull request by radai-rosenblatt**
this fixes a resource leak in generated client stubs.

HOWEVER. in my machine it magically succeeds to connect, about once every 100k attempts.
please verify this only happens to me ...

### CODE REVIEW

**Comment by radai-rosenblatt** [comment]
the test in this pull request triggers this grizzly bug - https://java.net/jira/browse/GRIZZLY-1807

**Comment by kofemann** [comment]
ok to test

File: oncrpc4j-rpcgen/src/test/java/org/dcache/oncrpc4j/rpcgen/TestUtil.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package org.dcache.oncrpc4j.rpcgen;
```
Comment (kofemann) [inline oncrpc4j-rpcgen/src/test/java/org/dcache/oncrpc4j/rpcgen/TestUtil.java]: please use Throwables.getRootCause() provided bu guava library

**Comment by radai-rosenblatt** [comment]
ok, switched to guava

**Comment by kofemann** [comment]
thanks!

**Pull request merged**